extensions/blocks/recurring-payments/edit.jsx

```diff
@@ -358,11 +358,14 @@ class MembershipsButtonEdit extends Component {
 				{ this.state.shouldUpgrade && (
 					<Placeholder
 						icon={ <BlockIcon icon={ icon } /> }
-						label={ __( 'Memberships', 'jetpack' ) }
+						label={ __( 'Recurring Payments', 'jetpack' ) }
 						notices={ notices }
 					>
 						<div className="components-placeholder__instructions wp-block-jetpack-membership-button">
-							{ __( "You'll need to upgrade your plan to use the Membership Button.", 'jetpack' ) }
+							{ __(
+								"You'll need to upgrade your plan to use the Recurring Payment Button.",
```

Comment:
Should it be payment**s**?

```suggestion
								"You'll need to upgrade your plan to use the Recurring Payments Button.",
```

Same for the other instances below.